forecast/preprocessor.py

```diff
@@ -347,14 +356,17 @@ def clock(label):
 
         print("Extract queries: ", end="", flush=True)
         df["query_raw"] = self._extract_query(df["message"])
+        df.drop(columns=["message"], inplace=True)
```

Comment:
I kind of liked the fact that we were only ever augmenting the data, and never dropping columns.
Otherwise we could and should have dropna'd inplace much earlier in the pipeline.

Because I expect this to be done offline in batches, the memory bloat is OK IMO.
Curious to know if you have thoughts there.

Reply:
This was originally done to control the bloat, is it okay to leave these in?

Reply:
I'm guessing you see significant memory savings for doing so, then? If so, sure. The main thing we lose is debuggability.